fs/dafsclient/rmtclient.hpp

```diff
@@ -63,7 +63,7 @@ extern DAFSCLIENT_API int getDafsInfo(ISocket * socket, unsigned level, StringBu
 extern DAFSCLIENT_API void setDafsEndpointPort(SocketEndpoint &ep);
 extern DAFSCLIENT_API void setDafsLocalMountRedirect(const IpAddress &ip,const char *dir,const char *mountdir);
 extern DAFSCLIENT_API ISocket *connectDafs(SocketEndpoint &ep, unsigned timeoutms, const IPropertyTree *service); // NOTE: might alter ep.port if configured for multiple ports ...
-extern DAFSCLIENT_API ISocket *checkSocketSecure(ISocket *socket);
+extern DAFSCLIENT_API ISocket *checkSocketSecure(ISocket *socket, unsigned timeoutms);
```

Comment:
cpp has unsigned timeoutms = DEFAULT_CONNECT_TIME, I suspect default should be here.